react/features/welcome/components/BlankPage.native.js

```diff
@@ -40,12 +49,34 @@ class BlankPage extends Component<Props> {
      * @returns {ReactElement}
      */
     render() {
+        const { _styles } = this.props;
+
         return (
-            <LocalVideoTrackUnderlay>
-                <NetworkActivityIndicator />
-            </LocalVideoTrackUnderlay>
+            <View
+                style = { [
+                    styles.blankPageWrapper,
+                    _styles.loadingOverlayWrapper
+                ] }>
+                <LoadingIndicator
+                    color = { _styles.indicatorColor }
+                    size = 'large' />
+            </View>
         );
     }
 }
 
-export default connect()(BlankPage);
+/**
+ * Maps part of the Redux state to the props of this component.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {{
+ *     _styles: StyleType
```

Comment:
I recently started to use just `Props` here, because these lists are not linted and get outdated very quickly

Reply:
Good catch. Will fix.